README.md

```diff
@@ -409,6 +409,8 @@ class MyExec(Executor):
                 docs.embedding = model_inference(docs.tensors)
 ```
 
+Jaeger or any other distributed tracing tool can be integrated to collect and visualize request-level and application level service operation attributes to analyze the request-response lifecycle, application behavior and performance.
```

Comment:
```suggestion
You can integrate Jaeger or other distributed tracing tools to collect and visualize request-level and application level service operation attributes. This helps you analyze request-response lifecycle, application behavior and performance.
```

- Stick to active tense. (i.e. "you can integrate foo" not "foo can be integrated") so it's clear WHO can do what
- Use short sentences